nemo_skills/inference/server/serve_nemo_asr.py

```
@@ -0,0 +1,358 @@
+                start_time = time.time()
+                hypotheses = self.model.transcribe(
+                    [tmp_path],
+                    batch_size=1,
+                    return_hypotheses=True,
+                    timestamps=enable_timestamps
+                )
+                inference_time = time.time() - start_time
+
+                # Extract transcription
+                if len(hypotheses) == 0:
+                    raise RuntimeError("No transcription returned from model")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)